Sources/Playbook/Components/Nav/NavCatalog.swift

```diff
@@ -277,7 +278,7 @@ struct NavCatalog: View {
       .padding(Spacing.medium)
     }
     .background(Color.background(Color.BackgroundColor.light))
-    .navigationTitle("Nav")
+    .navigationTitle("Nav Bar")
```

Comment:
Hey. Is this change part of this story? Is it suppose to be changed?

Reply:
Yes, that was from when I thought it was supposed to be part of the nav kit. I changed the top segment picker to have " Tab bar" and saw forgot to change it back to nav bar